buildspec.yml

```diff
@@ -0,0 +1,35 @@
+version: 0.2
+run-as: root
+
+phases:
+  install:
+    commands:
+      - echo Entered the install phase...
+      - nohup /usr/local/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2 &
+      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
+
+  pre_build:
+    commands:
+      - echo Entered the pre_build phase...
+
+  build:
+    commands:
+      - echo Entered the build phase...
+      - echo Build started on `date`
+    finally:
+      - echo This always runs even if the build command fails
+
+  post_build:
+    commands:
+      - echo Entered the post_build phase...
+      - echo Build completed on `date`
+# cache:
+#   paths:
+#     - "/root/.cache"
+#     - "/root/.yarn/**/*"
+#     - "/root/.npm/**/*"  # npm cache
+#     - "/root/.gradle/**/*"  # Gradle cache
+#     - "/root/.m2/**/*"  # Maven cache
+#     - "/root/.docker"  # Docker cache
+#     - "/root/.gem/**/*"  # Gem cache
+#     - "/root/go/pkg/mod/**/*"  # Go modules cache
```

Comment:
maybe remove this? 🗑️ 💭

Reply:
I may turn them back on, I'd like to keep it in case I do.

Reply:
I need to check backend vs front-end build times, I think the front-end is building about 1 minutes faster..